### PR TITLE
Add "HTTP 464 error with AWS" to the FAQ

### PR DESCRIPTION
**Pull request by timostamm**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -171,7 +171,23 @@ mappings for your backend services. You can work around this problem using the
...
+request, or for a web browser making a CORS preflight `OPTIONS` request.
+
+If you only need the Connect protocol, you can simply change your target group 
+to HTTP/2.
+
+If you need both the Connect and gRPC protocols, you can use two target groups:
+Route HTTP GET requests and anything with the `application/proto`, `application/json`, 
+`application/connect+proto`, or `application/connect+json` Content-Types to the 
+HTTP/2 target group. Route anything else to the gRPC target group.
+
```
Comment (rebello95) [inline docs/faq.md:187]: Nit: extra new line

File: docs/faq.md [hunk 2]
```diff
@@ -171,7 +171,23 @@ mappings for your backend services. You can work around this problem using the
 `cluster_tag` property, as described in
 [emissary-ingress/emissary#3112](https://github.com/emissary-ingress/emissary/issues/3112).
 
+### HTTP 464 error with AWS
+
+If you're using the [Application Load Balancer support for gRPC][alp-aws-grpc],
```
Comment (rebello95) [inline docs/faq.md:176]:
```suggestion
If you're using the AWS [Application Load Balancer support for gRPC][alp-aws-grpc], 
```

File: docs/faq.md [hunk 3]
```diff
@@ -171,7 +171,23 @@ mappings for your backend services. You can work around this problem using the
 `cluster_tag` property, as described in
 [emissary-ingress/emissary#3112](https://github.com/emissary-ingress/emissary/issues/3112).
 
+### HTTP 464 error with AWS
+
+If you're using the [Application Load Balancer support for gRPC][alp-aws-grpc], 
+you will likely see an HTTP error response with code 464 for a Connect `GET` 
+request, or for a web browser making a CORS preflight `OPTIONS` request.
```
Comment (rebello95) [inline docs/faq.md:178]: Would it be useful to link out to an explanation of when Connect does these actions? (Such as how to opt in to GET support)

**Comment by rebello95** [comment]
cc @akshayjshah

File: docs/faq.md (hunk 2 above)
Comment (timostamm) [reply docs/faq.md:176]: I'd argue the context should be clear from the headline, but don't mind making it extra clear.

File: docs/faq.md (hunk 3 above)
Comment (timostamm) [reply docs/faq.md:178]: I considered linking to:
- the blog post
- protocol definition for Unary-Get-Request
- Go / Get Requests and Caching
- Node.js / Get Requests and Caching

None of them seems like a good fit to me.

File: docs/faq.md (hunk 1 above)
Comment (rebello95) [reply docs/faq.md:187]: (Should be removed)

File: docs/faq.md (hunk 3 above)
Comment (rebello95) [reply docs/faq.md:178]: Alright, probably okay to leave it out then

File: docs/faq.md [hunk 4]
```diff
@@ -171,7 +171,23 @@ mappings for your backend services. You can work around this problem using the
...
+
+If you only need the Connect protocol, you can simply change your target group 
+to HTTP/2.
+
+If you need both the Connect and gRPC protocols, you can use two target groups:
+Route HTTP GET requests and anything with the `application/proto`, `application/json`, 
+`application/connect+proto`, or `application/connect+json` Content-Types to the 
+HTTP/2 target group. Route anything else to the gRPC target group.
```
Comment (akshayjshah) [inline docs/faq.md:186]: I'm not sure how to keep the text short, but I found this explanation a little _too_ short. Even if the user needs support for gRPC, they can continue to use an HTTP/2 target group - as far as I know, their RPCs will work just fine. However, they'll give up support for the gRPC-specific add-on features that gRPC target groups provide.

Comment (timostamm) [reply docs/faq.md]: Good call. I expanded it to clarify.

**Review by akshayjshah — APPROVED** [review]

**Pull request merged**